packages/dashboard/src/components/csvDownloadButton/useViewportData.ts

```diff
@@ -62,18 +62,18 @@ export const useViewportData = ({
         );
 
         const flatDataPoint: CSVDownloadObject = {
-          id,
+          assetName: describedAssetsMap[assetPropId[0]]?.assetName,
+          propertyName: describedModelProperty?.name,
+          propertyAlias: isUnmodeledData ? id : describedModelProperty?.alias,
           value: yValue,
           unit,
-          timestamp: formatDate(new Date(xValue)),
-          aggregationType,
+          timestamp: new Date(xValue).toISOString(),
+          aggregationType: aggregationType ?? 'raw',
```

Comment:
If we want to align with the SiteWise API, 'raw' isn't a concept. Why not leave the field empty?

Reply:
oh yes, and resolution = 0 already denotes raw data 
will update